subprojects/docs/src/docs/userguide/core-plugins/pmd_plugin.adoc

```diff
@@ -80,3 +80,20 @@ include::sample[dir="snippets/codeQuality/codeQuality/kotlin",files="build.gradl
 ====
 
 See the link:{groovyDslPath}/org.gradle.api.plugins.quality.PmdExtension.html[PmdExtension] class in the API documentation.
+
+[[sec:pmd_conf_threads]]
+=== Parallel analysis
+
+You can configure the number of threads to be used by PMD for running its analysis.
+
+====
+include::sample[dir="snippets/codeQuality/codeQuality/groovy",files="build.gradle[tags=pmd-threads]"]
+include::sample[dir="snippets/codeQuality/codeQuality/kotlin",files="build.gradle.kts[tags=pmd-threads]"]
+====
+
+[NOTE]
+====
+This configuration is internal to PMD and is not linked to <<performance#parallel_execution,the number of workers>> used by Gradle.
```

Comment:
IMHO this is a bit vague. We could maybe clearly state instead how the two thread counts interact, if they multiply or not.

Reply:
This is covered by the sentences in that block